src/TextOverflow/TextOverflow.js

```diff
@@ -9,7 +9,11 @@ import ResizeSensor from '../ResizeSensor/ResizeSensor';
 
 export const Props = {
   children: t.maybe(t.Function),
-  label: t.String,
+  label: t.maybe(t.union([t.String, t.Number])),
+  popover: t.maybe(t.interface({
```

Comment:
can you add below a description for the new `popover` prop?

Reply:
Done 👍